libV2/helpers/collection/generateSkeletionTreeFromOpenAPI.js

```diff
@@ -0,0 +1,261 @@
+           * include deprecated handling.
+           * If true, add in the postman collection. If false ignore the request.
+           */
+          if (!includeDeprecated && data.deprecated) {
+            return;
+          }
+
+          tree.setNode(`path:${pathSplit[0]}:${method}`, {
+            type: 'request',
+            meta: {
+              path: path,
+              method: method,
+              pathIdentifier: pathSplit[0]
+            },
+            data: {}
+          });
+
+          tree.setEdge('root:collection', `path:${pathSplit[0]}:${method}`);
+        });
+      }
+
+      else {
+        _.forEach(pathSplit, function (p, index) {
+          let previousPathIdentified = pathSplit.slice(0, index).join('/'),
+            pathIdentifier = pathSplit.slice(0, index + 1).join('/');
+
+          /**
+           * Always first try to find the node if it already exists.
+           * if yes, bail out nothing is needed to be done.
+           */
+          if (tree.hasNode(`path:${pathIdentifier}`)) {
+            return;
+          }
+
+          else {
+            tree.setNode(`path:${pathIdentifier}`, {
+              type: 'folder',
+              meta: {
+                path: p,
+                pathIdentifier: pathIdentifier
+              },
+              data: {}
+            });
+
+            /**
+             * If index is 0, this means that we are on the first level.
+             * Hence it is folder/request to be added on the first level
+             *
+             * If after the split we have more than one paths, then we need
+             * to add to the previous node.
+             */
+            tree.setEdge(index === 0 ? 'root:collection' : `path:${previousPathIdentified}`, `path:${pathIdentifier}`);
+          }
+        });
+
+        /**
+         * Now for all the methods present in the path, add the request nodes.
+         */
+
+        _.forEach(methods, function (data, method) {
+          if (!ALLOWED_HTTP_METHODS[method]) {
+            return;
+          }
+
+          /**
+           * include deprecated handling.
+           * If true, add in the postman collection. If false ignore the request.
+           */
+          if (!includeDeprecated && data.deprecated) {
+            return;
+          }
+
+          // join till the last path i.e. the folder.
+          let previousPathIdentified = pathSplit.slice(0, (pathSplit.length)).join('/'),
+            pathIdentifier = `${pathSplit.join('/')}:${method}`;
+
+          tree.setNode(`path:${pathIdentifier}`, {
+            type: 'request',
+            data: {},
+            meta: {
+              path: path,
+              method: method,
+              pathIdentifier: pathIdentifier
+            }
+          });
+
+          tree.setEdge(`path:${previousPathIdentified}`, `path:${pathIdentifier}`);
+        });
+      }
+    });
+
+    return tree;
+  },
+
+  _generateTreeFromTags = function (openapi, { includeDeprecated }) {
```

Comment:
We are not generating collection from `Tags` correctly. No description of tag is used in below code. I compared the output with v1, see below comparison for [this file](https://github.com/postmanlabs/openapi-to-postman/blob/31874394c6a745d2aa2b171ac7434035055ae343/test/data/valid_openapi/petstore-detailed.yaml#L1).

V2 output:
![image](https://user-images.githubusercontent.com/16756296/223024779-7d185698-cd86-44a2-818f-edc782e9b6ad.png)


V1 output:
![image](https://user-images.githubusercontent.com/16756296/223024798-2489a875-8c73-421e-bf10-6cc65744db7e.png)

Also, we've skipped tests around folderStratagy value `Tags` in V2 tests. can we add it?